test/unit/core/reducers/data/personalization.ts

```diff
@@ -0,0 +1,190 @@
+import { Actions, Store } from '../../../../../src/core';
+import * as personalization from '../../../../../src/core/reducers/data/personalization';
+import suite from '../../../_suite';
+
+suite('personalization', ({ expect, spy, stub }) => {
+  describe('updatePersonalization()', () => {
+    const state: Store.Personalization = {
+      biasing: <any>{ a: 1 }
+    };
+    const updatePersonalization = personalization.default;
```

Comment:
take a look at the other reducer test for the pattern of how to import and name this, 
should use the `import` syntax to get both:
```
import reducer, * as personalization from '../wherever';
```